core/src/lib/util.h

```diff
@@ -89,4 +89,11 @@ std::string TPAsString(const std::chrono::system_clock::time_point& tp);
 regex_t* StringToRegex(const char* input);
 void to_lower(std::string& s);
 
+// see N3876 / boost::hash_combine
+static inline std::size_t hash_combine(std::size_t seed, std::size_t hash)
```

Comment:
sorry to bug you again. `static inline` is not required in C++ and will in fact provide a slightly worse result than a plain `inline`.